packages/story-editor/src/app/story/effects/useLoadStory.js

```diff
@@ -58,51 +56,33 @@ function useLoadStory({ storyId, shouldLoad, restore, isDemo }) {
           preview_link: previewLink,
           edit_link: editLink,
           embed_post_link: embedPostLink,
-          _embedded: embedded = {},
-          _links: links = {},
+          author,
+          capabilities = {
+            hasPublishAction: false,
+            hasAssignAuthorAction: false,
+          },
+          lock_user,
+          featured_media,
+          publisher_logo,
+          taxonomies,
+          terms,
         } = post;
 
-        const capabilities = {
-          hasPublishAction: Object.prototype.hasOwnProperty.call(
-            links,
-            'wp:action-publish'
-          ),
-          hasAssignAuthorAction: Object.prototype.hasOwnProperty.call(
-            links,
-            'wp:action-assign-author'
-          ),
-        };
-
-        const author = {
-          id: embedded?.author?.[0].id || 0,
-          name: embedded?.author?.[0].name || '',
-        };
-
-        const lockUser = {
-          id: embedded?.['wp:lockuser']?.[0].id || 0,
-          name: embedded?.['wp:lockuser']?.[0].name || '',
-          avatar: embedded?.['wp:lockuser']?.[0].avatar_urls?.['96'] || '',
+        const defaultMedia = {
+          id: 0,
+          height: 0,
+          width: 0,
+          url: '',
         };
 
-        const featuredMedia = {
-          id: embedded?.['wp:featuredmedia']?.[0].id || 0,
-          height:
-            embedded?.['wp:featuredmedia']?.[0]?.media_details?.height || 0,
-          width: embedded?.['wp:featuredmedia']?.[0]?.media_details?.width || 0,
-          url: embedded?.['wp:featuredmedia']?.[0]?.source_url || '',
+        const lockUser = lock_user ?? {
+          id: 0,
+          name: '',
+          avatar: '',
         };
 
-        const publisherLogo = {
-          id: embedded?.['wp:publisherlogo']?.[0].id || 0,
-          height:
-            embedded?.['wp:publisherlogo']?.[0]?.media_details?.height || 0,
-          width: embedded?.['wp:publisherlogo']?.[0]?.media_details?.width || 0,
-          url: embedded?.['wp:publisherlogo']?.[0]?.source_url || '',
-        };
-
-        const taxonomies =
-          links?.['wp:term']?.map(({ taxonomy }) => taxonomy) || [];
-        const terms = embedded?.['wp:term'] || [];
+        const featuredMedia = featured_media ?? defaultMedia;
+        const publisherLogo = publisher_logo ?? defaultMedia;
```

Comment:
`transformGetStoryResponse` ensures that this object always exists, so no need for this check here